tests/Agent/IntegrationTests/IntegrationTests/HttpClientInstrumentation/NetCore/HttpClientInstrumentationNetCore.cs

```diff
@@ -55,7 +55,8 @@ public void Test()
                 new Assertions.ExpectedMetric { metricName = @"External/www.bing.com/Stream/GET", callCount = 1 },
                 new Assertions.ExpectedMetric { metricName = @"External/www.google.com/Stream/GET", metricScope = @"WebTransaction/MVC/Home/HttpClient", callCount = 1 },
                 new Assertions.ExpectedMetric { metricName = @"External/www.yahoo.com/Stream/GET", metricScope = @"WebTransaction/MVC/Home/HttpClient", callCount = 1 },
-                new Assertions.ExpectedMetric { metricName = @"External/www.bing.com/Stream/GET", metricScope = @"WebTransaction/MVC/Home/HttpClientTaskCancelled", callCount = 1 }
+                new Assertions.ExpectedMetric { metricName = @"External/www.bing.com/Stream/GET", metricScope = @"WebTransaction/MVC/Home/HttpClientTaskCancelled", callCount = 1 },
+                new Assertions.ExpectedMetric { metricName = @"Supportability/SpanEvent/TotalEventsSeen", CallCountAllHarvests = 9 }
```

Comment:
Was this change just to improve the base test?

Reply:
Correct, this is to make sure we won't get extra spans due to double instrumentation on the HttpClient calls prior to the fix.